### PR TITLE
Adds AppVeyor CI config for Windows

### PR DESCRIPTION
**Pull request by rasbt**
## What does this PR do?

Adds a config file to test TPOT on Windows via AppVeyor CI

### CODE REVIEW

**Comment by rhiever** [comment]
Awesome, thanks for this @rasbt! It looks like AppVeyor isn't integrated into GitHub yet?

**Comment by rhiever** [comment]
Ohhh sweet, it IS integrated into GitHub! Looks like we're [failing](https://ci.appveyor.com/project/rhiever/tpot/build/1.0.1) on AppVeyor though.

**Comment by rhiever** [comment]
Ohhh sweet, it IS integrated into GitHub! Looks like we're [failing](https://ci.appveyor.com/project/rhiever/tpot/build/1.0.1) on AppVeyor though because of the XGBoost install. Not surprised there.

**Comment by rasbt** [comment]
The setup seems to work in general:

screen shot 2017-06-02 at 2 27 31 pm

There seems to be an issue with XGBoost and pip though. Do you know if XGBoost even supported by Windows? Or is there some trick for pip-installing it on Windows?

**Comment by rhiever** [comment]
Last I checked, XGBoost was practically impossible to install on Windows. But that's been a long time because I don't use Windows. Maybe @terrytangyuan can comment?

**Comment by terrytangyuan** [comment]
I don't use Windows either. @phunterlau maintains the Windows binaries, @phunterlau could you share some instruction/tricks here?

**Comment by rasbt** [comment]
Does TPOT currently run without having XGBoost installed? In this case, I'd suggest to remove it from the Windows tests

**Comment by phunterlau** [comment]
@terrytangyuan sorry I didn't maintain the windows binary for XGBoost: there is no official. We have the plan that, I talked with MXNet's windows binary maintainer for reusing his expertise on making an XGboost windows binary in PyPI (plus a variant with GPU tree building), but it may take a little bit more time. Let me check with him about the progress. 

Meanwhile, please follow https://www.ibm.com/developerworks/community/blogs/jfp/entry/Installing_XGBoost_For_Anaconda_on_Windows?lang=en to build XGBoost in windows.

**Comment by rhiever** [comment]
TPOT can indeed run without XGBoost, and I don't think any of the unit tests should be affected by not having it.

**Comment by rhiever** [comment]
Thank you for clarifying @terrytangyuan and @phunterlau!

**Comment by rasbt** [comment]
Should be fine now. Not sure what's going on with the test coverage decrease though -- maybe it's somehow accounting for the fact that XGBoost related code is not being tested in AppVeyor (however, the travis tests are unchanged)

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON_VERSION: 3.6
```
Comment (rhiever) [inline .appveyor.yml]: Can we add multiple Python versions here, e.g., 2.7 as well?

**Comment by rasbt** [comment]
sure

**Comment by rasbt** [comment]
seems to work now

**Comment by rhiever** [comment]
Awesome! Thanks so much @rasbt.

**Pull request merged**